socket-saver.user.js

```diff
@@ -0,0 +1,27 @@
+// ==UserScript==
```

Comment:
That's the plan.

> I will merge the two in a single script if you think it is working fine.
(in the original pr message :P)

I just want to have @rlemon test it and make sure it works -_-